src/index.js

```diff
@@ -219,6 +219,12 @@ module.exports = {
   get SafeAreaInsetsManager() {
     return require('./nativeComponents').SafeAreaInsetsManager;
   },
+  get KeyboardAwareInsetsView() {
```

Comment:
why one is commented and the other isn't..

Reply:
Also didn't we decide to export those under the `Keyboard` namespace

Reply:
Not sure, the first might be required because it is being used.
Both will be fixed in one of the other PRs.